### ISSUE TITLE
This is question not issue

### ISSUE THREAD

**Issue by minhquan1313** [NekoShooter/Vany#1]
Hi dev, I find this library really interesting. I just have a question, do you plan to write an English version with Typescript implement? That would be great! And another question is whether Vanie has any method to prevent dragging window out of view? Like "limitMovement" from [windrag](https://github.com/Axorax/windrag.js) library?

**Comment by NekoShooter** [comment]
Yes, I have planned to make the English version, but since I don't know English very well, it will likely have spelling and syntax errors. Regarding the implementation with TypeScript, I haven't considered it yet, but I might make it a priority. As for the function to prevent the window from moving, it doesn't exist, but it's a great idea, so thank you. I'll start working on that method.

I appreciate the feedback, my friend 😉.

**Comment by NekoShooter** [comment]
I will start creating the English version of this library as soon as possible. The method to limit movement, called `"limitarMovimiento,"` has already been created, and I have also submitted a **`PR`** on `GitHub`. For now, on `NPM`, this method will be included in version 0.0.5 along with the other improvements you mentioned, as well as some additional ones that I am preparing.

``` JavaScript
import Vanie from "vanie"

const root= document.getElementById('root');

const window_noMove = new Vanie('windows-claro');
const window = new Vanie('linux-oscuro');

window_noMove.padre = window .padre =root;

//true: no move, false: move
window_noMove.limitarMovimiento(true);

window_noMove.abrir();
window.abrir();

window_noMove.lienzo.style.backgroundColor = '#00000077';
window.lienzo.style.backgroundColor = '#059b9add';
```

<p align="center"><a href="#"><img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaWZ2bmp5eHVxeXFtNW12dDgxbGU3bGZ6a21qZHZnaTZteWJjb3JseiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/qSm75pkks62xEP2Iib/giphy.gif" alt="gif animado que muestra el resultado del codigo usado en el ejemplo de limitarMovimiento"></a></p>
